install.ps1

```diff
@@ -0,0 +1,56 @@
+If (-NOT ([Security.Principal.WindowsPrincipal] [Security.Principal.WindowsIdentity]::GetCurrent()).IsInRole(`
```

Comment:
I think it would be better if you can somehow detect and respawn in administrator mode. Instead of just erroring out.